### ISSUE TITLE
Compatibility issue with jeduan/cordova-plugin-facebook4

### ISSUE THREAD

**Issue by jeffwray** [arnesson/cordova-plugin-firebase#1015]
**Describe the bug**
On iOS, I've been using the cordova-plugin-facebook4 for over two years without problems. As soon as I implemented cordova-plugin-firebase the Facebook login process broke, it seems when Facebook calls back the app url custom schema cordova-plugin-firebase may be doing something with that callback, instead of passing it to the Facebook process.eing reported.

**Plugin Version**
cordova-plugin-actionsheet 2.3.3 "ActionSheet"
cordova-plugin-battery-status 1.2.5 "Battery"
cordova-plugin-camera 2.4.1 "Camera"
cordova-plugin-compat 1.2.0 "Compat"
cordova-plugin-device 1.1.7 "Device"
cordova-plugin-dialogs 1.3.4 "Notification"
cordova-plugin-file 4.3.3 "File"
cordova-plugin-firebase 2.0.5 "Google Firebase Plugin"
cordova-plugin-geolocation 2.4.3 "Geolocation"
cordova-plugin-globalization 1.0.9 "Globalization"
cordova-plugin-inappbrowser 1.7.2 "InAppBrowser"
cordova-plugin-media-capture 1.4.3 "Capture"
cordova-plugin-network-information 1.3.4 "Network Information"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-vibration 2.1.6 "Vibration"
cordova-plugin-whitelist 1.3.3 "Whitelist"
phonegap-plugin-barcodescanner 8.0.1 "BarcodeScanner"
uk.co.workingedge.phonegap.plugin.launchnavigator 4.2.2 "Launch Navigator"

**Comment by globules-io** [comment]
@jeffwray I do not think it is the issue because I am using cordova-plugin-facebook4 + cordova-plugin-firebase and my login with Facebook works as expected.

**Comment by guylando** [comment]
@jeffwray Try this fork which is supposed to fix it: https://github.com/guylando/cordova-plugin-facebook4
See: https://github.com/jeduan/cordova-plugin-facebook4/issues/750
